Common/File/FileUtil.cpp

```diff
@@ -652,7 +652,9 @@ bool CreateFullPath(const Path &path) {
 	}
 
 	std::vector<std::string_view> parts;
-	SplitString(diff, '/', parts);
```

Comment:
hm, was this really a problem?

Reply:
Yes, I tested it.

Reply:
Ok. though, I might revert this change and fix SplitString instead in a followup..

Reply:
OK, but I don't see what is wrong with `SplitString`. It is called with an empty string and produces a vector with one empty string. That seems correct.

Reply:
Then how was it possibly a problem?

Reply:
Oh wait, I see how that breaks in the following loop.